FE/src/app/BigImportNgZorro.ts

```diff
@@ -0,0 +1,149 @@
+import {NzRadioModule} from 'ng-zorro-antd/radio';
+import {NzRateModule} from 'ng-zorro-antd/rate';
+import {NzResultModule} from 'ng-zorro-antd/result';
+import {NzSelectModule} from 'ng-zorro-antd/select';
+import {NzSkeletonModule} from 'ng-zorro-antd/skeleton';
+import {NzSliderModule} from 'ng-zorro-antd/slider';
+import {NzSpinModule} from 'ng-zorro-antd/spin';
+import {NzStatisticModule} from 'ng-zorro-antd/statistic';
+import {NzStepsModule} from 'ng-zorro-antd/steps';
+import {NzSwitchModule} from 'ng-zorro-antd/switch';
+import {NzTableModule} from 'ng-zorro-antd/table';
+import {NzTabsModule} from 'ng-zorro-antd/tabs';
+import {NzTagModule} from 'ng-zorro-antd/tag';
+import {NzTimePickerModule} from 'ng-zorro-antd/time-picker';
+import {NzTimelineModule} from 'ng-zorro-antd/timeline';
+import {NzToolTipModule} from 'ng-zorro-antd/tooltip';
+import {NzTransferModule} from 'ng-zorro-antd/transfer';
+import {NzTreeModule} from 'ng-zorro-antd/tree';
+import {NzTreeViewModule} from 'ng-zorro-antd/tree-view';
+import {NzTreeSelectModule} from 'ng-zorro-antd/tree-select';
+import {NzTypographyModule} from 'ng-zorro-antd/typography';
+import {NzUploadModule} from 'ng-zorro-antd/upload';
+import {NzResizableModule} from 'ng-zorro-antd/resizable';
+import {NzPipesModule} from 'ng-zorro-antd/pipes';
+
+@NgModule({
+  exports: [
+    NzAffixModule,
+    NzAlertModule,
+    NzAnchorModule,
+    NzAutocompleteModule,
+    NzAvatarModule,
+    NzBackTopModule,
+    NzBadgeModule,
+    NzButtonModule,
+    NzBreadCrumbModule,
+    NzCalendarModule,
+    NzCardModule,
+    NzCarouselModule,
+    NzCascaderModule,
+    NzCheckboxModule,
+    NzCollapseModule,
+    NzCommentModule,
+    NzDatePickerModule,
+    NzDescriptionsModule,
+    NzDividerModule,
+    NzDrawerModule,
+    NzDropDownModule,
+    NzEmptyModule,
+    NzFormModule,
+    NzGridModule,
+    NzI18nModule,
+    NzIconModule,
+    NzImageModule,
+    NzInputModule,
+    NzInputNumberModule,
+    NzLayoutModule,
+    NzListModule,
+    NzMentionModule,
+    NzMenuModule,
+    NzMessageModule,
+    NzModalModule,
+    NzNoAnimationModule,
+    NzNotificationModule,
+    NzPageHeaderModule,
+    NzPaginationModule,
+    NzPopconfirmModule,
+    NzPopoverModule,
+    NzProgressModule,
+    NzRadioModule,
+    NzRateModule,
+    NzResultModule,
+    NzSelectModule,
+    NzSkeletonModule,
+    NzSliderModule,
+    NzSpinModule,
+    NzStatisticModule,
+    NzStepsModule,
+    NzSwitchModule,
+    NzTableModule,
+    NzTabsModule,
+    NzTagModule,
+    NzTimePickerModule,
+    NzTimelineModule,
+    NzToolTipModule,
+    NzTransButtonModule,
+    NzTransferModule,
+    NzTreeModule,
+    NzTreeViewModule,
+    NzTreeSelectModule,
+    NzTypographyModule,
+    NzUploadModule,
+    NzWaveModule,
+    NzResizableModule,
+    NzPipesModule
+  ]
+})
+export class ZorroModule {
+
+}
+
+// This file is used so that I don't have to import each of the components whenever I want to use them,
```

Comment:
If you are not using all of the components they still will be in the findle bundle increasing the bundle size for no reason